functions/icons.zsh

```diff
@@ -213,22 +213,22 @@ case $POWERLEVEL9K_MODE in
       VCS_COMMIT_ICON                $'\uE729 '             # 
       VCS_BRANCH_ICON                $'\uF126 '             # 
       VCS_REMOTE_BRANCH_ICON         $'\uE728 '             # 
-      VCS_GIT_ICON                   $'\uF113 '             # 
+      VCS_GIT_ICON                   $''                    # 
```

Comment:
Btw. the official GIT icon is in NerdFonts:
![bildschirmfoto 2018-04-21 um 15 37 46](https://user-images.githubusercontent.com/1544760/39084759-325b8dae-457a-11e8-9383-6ca9a0285c40.png)